dotnet/src/webdriver/Internal/PortUtilities.cs

```diff
@@ -24,7 +24,7 @@ namespace OpenQA.Selenium.Internal
     /// <summary>
     /// Encapsulates methods for working with ports.
     /// </summary>
-    internal static class PortUtilities
+    public static class PortUtilities
```

Comment:
Keep it internal, already implemented in `trunk`.

Reply:
I'll rebase on trunk